<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-6TDNORL" name="TestScenario.firstproj" tests="10" failures="0" timestamp="31 Oct 2017 12:35:30 GMT" time="231.480" errors="1">
  <testcase name="loginPage" time="3.183" classname="TestScenario.firstproj"/>
  <testcase name="clickOnContact" time="12.384" classname="TestScenario.firstproj"/>
  <testcase name="clickingNew" time="1.327" classname="TestScenario.firstproj"/>
  <testcase name="setValuesNewPage" time="11.520" classname="TestScenario.firstproj"/>
  <testcase name="read_Contact_Name" time="0.263" classname="TestScenario.firstproj"/>
  <testcase name="writeInSheet" time="0.931" classname="TestScenario.firstproj"/>
  <testcase name="readFromExcel" time="0.039" classname="TestScenario.firstproj"/>
  <testcase name="clickOnContact2" time="0.637" classname="TestScenario.firstproj"/>
  <testcase name="clickingGoforListView" time="1.705" classname="TestScenario.firstproj"/>
  <testcase name="navigatingToRecord" time="199.491" classname="TestScenario.firstproj">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at TestScenario.listView.navigatingToRecord(listView.java:44)
	at TestScenario.firstproj.navigatingToRecord(firstproj.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- navigatingToRecord -->
</testsuite> <!-- TestScenario.firstproj -->
